src/Sarif.Converters/Sarif.Converters.csproj

```diff
@@ -1,92 +1,48 @@
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Core" />
-    <Reference Include="System.IO.Compression" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
+  
   <ItemGroup>
-    <None Include="packages.config" />
-    <EmbeddedResource Include="Schemata\FxCopReport.xsd">
-      <SubType>Designer</SubType>
-    </EmbeddedResource>
+    <None Remove="Schemata\FxCopReport.xsd" />
   </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="AndroidStudioConverter.cs" />
-    <Compile Include="AndroidStudioProblem.cs" />
-    <Compile Include="AndroidStudioStrings.cs" />
-    <Compile Include="BuiltInConverterFactory.cs" />
-    <Compile Include="ClangAnalyzerConverter.cs" />
-    <Compile Include="CppCheckConverter.cs" />
-    <Compile Include="CppCheckError.cs" />
-    <Compile Include="CppCheckLocation.cs" />
-    <Compile Include="CppCheckStrings.cs" />
-    <Compile Include="Extensions.cs" />
-    <Compile Include="FileInfoFactory.cs" />
-    <Compile Include="FortifyConverter.cs" />
-    <Compile Include="FortifyFprConverter.cs" />
-    <Compile Include="FortifyFprStrings.cs" />
-    <Compile Include="FortifyIssue.cs" />
-    <Compile Include="FortifyPathElement.cs" />
-    <Compile Include="FortifyStrings.cs" />
-    <Compile Include="FxCopConverter.cs" />
-    <Compile Include="ConverterFactory.cs" />
-    <Compile Include="IgnoreOptions.cs" />
-    <Compile Include="PluginConverterFactory.cs" />
-    <Compile Include="Ref.cs" />
-    <Compile Include="ConverterResources.Designer.cs">
-      <DependentUpon>ConverterResources.resx</DependentUpon>
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-    </Compile>
-    <Compile Include="SemmleQLConverter.cs" />
-    <Compile Include="SparseReader.cs" />
-    <Compile Include="SparseReaderDispatchTable.cs" />
-    <Compile Include="StaticDriverVerifierConverter.cs" />
-    <Compile Include="ToolFileConverterBase.cs" />
-    <Compile Include="ToolFormat.cs" />
-    <Compile Include="ToolFormatConverter.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <EmbeddedResource Include="Schemata\FxCopReport.xsd" />
   </ItemGroup>
+
   <ItemGroup>
-    <ProjectReference Include="..\Sarif\Sarif.csproj">
-      <Project>{cc9b247e-7103-4bf7-bdab-6e625b3680a8}</Project>
-      <Name>Sarif</Name>
-    </ProjectReference>
+    <PackageReference Include="CsvHelper" Version="2.16.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Condition="'$(TargetFramework)' == 'net452'" Include="System.Collections.Immutable" Version="1.1.37" />
```

Comment:
The version of Sarif.Converters used in the SARIF Viewer needs this specific version of System.Collections.Immutable because the Visual Studio Shell expects it.